packages/neon-core/src/consts.ts

```diff
@@ -7,15 +7,17 @@ export enum MAGIC_NUMBER {
 }
 
 export enum NATIVE_CONTRACT_HASH {
-  NeoToken = "f61eebf573ea36593fd43aa150c055ad7906ab83",
-  GasToken = "70e2301955bf1e74cbb31d18c2f96972abadb328",
-  PolicyContract = "79bcd398505eb779df6e67e4be6c14cded08e2f2",
-  ManagementContract = "a501d7d7d10983673b61b7a2d3a813b36f9f0e43",
-  OracleContract = "8dc0e742cbdfdeda51ff8a8b78d46829144c80ee",
-  DesignationContract = "c0073f4c7069bf38995780c9da065f9b3949ea7a",
-  LedgerContract = "971d69c6dd10ce88e7dfffec1dc603c6125a8764",
-  RoleManagement = "597b1471bbce497b7809e2c8f10db67050008b02",
-  NameService = "a2b524b68dfe43a9d56af84f443c6b9843b8028c",
+  NeoToken = "ef4073a0f2b305a38ec4050e4d3d28bc40ea63f5",
+  GasToken = "d2a4cff31913016155e38e474a2c06d08be276cf",
+  PolicyContract = "cc5e4edd9f5f8dba8bb65734541df7a1c081c67b",
+  ManagementContract = "fffdc93764dbaddd97c48f252a53ea4643faa3fd",
+  OracleContract = "fe924b7cfe89ddd271abaf7210a80a7e11178758",
+  DesignationContract = "49cf4e5378ffcd4dec034fd98a174c5491e395e2", // renamed to RoleManagement

```

Comment:
You mean the unused contracts? I'd keep them because they can be used with the experimental package, but also with the `invokefunction` RPC call or other custom script people might want to build and send using `sendrawtransaction`.

Reply:
I mean the renamed contract. Feel free to just go ahead and merge anyway since it's just a nit